Co-expression network analysis using RNA-Seq data
-------------------------------------------------

- **Author**: V. Keith Hughitt
- **Date**: June 15, 2016

### Overview

In this presentation, I provide a simple overview of co-expression network
analysis, with an emphasis on the use of RNA-Seq data. A motivation for the use
of co-expression network analysis is provided and compared to other common
types of RNA-Seq analyses such as differential expression analysis and gene set
enrichment analysis. The use of adjacency matrices to represent networks is
explored for several different types of networks and a small synthetic dataset
is used to demonstrate each of the major steps in co-expression network
construction and module detection. The tutorial portion of the presentation
then applies some of these principles using a real dataset containing ~3000
genes, after filtering.

- **Presentation**: [http://khughitt.github.io/2016-iscb-dc-rsg-workshop-presentation/](http://khughitt.github.io/2016-iscb-dc-rsg-workshop-presentation/#1) 
- **Video**: [https://www.youtube.com/watch?v=OdqDE5EJSlA](https://www.youtube.com/watch?v=OdqDE5EJSlA)
- **Tutorial**: [tutorial/README.md](https://github.com/iscb-dc-rsg/2016-summer-workshop/blob/master/3B-Hughitt-RNASeq-Coex-Network-Analysis/tutorial/README.md)
- **Data**:
    - Files
        - [data/count_table.csv](data/count_table.csv)
        - [data/sample_metadata.csv](data/sample_metadata.csv)
    - Source: [Fernandes et al, 2016](http://mbio.asm.org/content/7/3/e00027-16.abstract)

